internal/page/supporter/login_callback.go

```diff
@@ -35,21 +35,31 @@ func LoginCallback(oneLoginClient LoginCallbackOneLoginClient, sessionStore sesh
 
 		session := &sesh.LoginSession{
 			IDToken: idToken,
-			Sub:     userInfo.Sub,
+			Sub:     "supporter-" + userInfo.Sub,
 			Email:   userInfo.Email,
 		}
 
-		if err := sesh.SetLoginSession(sessionStore, r, w, session); err != nil {
-			return err
-		}
-
 		ctx := page.ContextWithSessionData(r.Context(), &page.SessionData{SessionID: session.SessionID()})
 
-		_, err = organisationStore.Get(ctx)
+		organisation, err := organisationStore.Get(ctx)
 		if err == nil {
+			session.OrganisationID = organisation.ID
```

Comment:
We could extend session with `OrganisationName` too, then we can let that flow through to AppData so the header contains it. Having it not be "live" is an ok tradeoff for now?

Reply:
Yeah thats probably a good idea, I'll add in.